pkg/repository/aws/dynamo.go

```diff
@@ -93,12 +92,12 @@ func (s *Dynamo) GetDBItem(tableName string, pk string, sk string) (map[string]*
 		return nil, nil
 	}
 
-	// fmt.Println("result from query: ", result)
-	return result.Item, nil
+	err = dynamodbattribute.UnmarshalMap(result.Item, &crowdaction)
+	return &crowdaction, err
 }
 
-func (s *Dynamo) Query(tableName string, filterCond expression.ConditionBuilder, startFrom *utils.PrimaryKey) ([]models.CrowdactionData, error) {
-	// var crowdactions models.CrowdactionData
+func (s *Dynamo) Query(tableName string, filter string, startFrom *utils.PrimaryKey) ([]models.CrowdactionData, error) {
+	filterCond := expression.Name(filter).GreaterThan(expression.Value(utils.GetDateStringNow()))
```

Comment:
Can we be sure that it is always `> now` in the filter`?  
For "ended" this does not seem to be the case.  
Maybe these things should also be covered by unit tests?